homie-controller/src/lib.rs

```diff
@@ -422,7 +426,10 @@ impl HomieController {
                 // Add new properties.
                 for property_id in properties {
                     if !node.properties.contains_key(property_id) {
-                        node.add_property(Property::new(property_id));
+                        let mut new_prop = Property::new(property_id);
+                        let key = format!("{}/{}/{}", device_id, node_id, property_id);
+                        new_prop.value = early_property_values.get(&key).cloned();
```

Comment:
could we get away with calling .remove() here?

Reply:
Absolutely. Thanks!